JitsiConference.js

```diff
@@ -775,6 +775,18 @@ JitsiConference.prototype.setLastN = function(lastN) {
 };
 
 /**
+ * Checks if the participant given by participantId is currently included in
+ * the last N.
+ * @param {string} participantId the identifier of the participant we would
+ * like to check
+ * @return {boolean} {true} if the participant with id is in the last N set or
```

Comment:
I'm not sure if {true}/{false} is the right syntax. I think it's used only to indicate a type (but I may be wrong)